io_scene_niftools/modules/nif_export/armature/__init__.py

```diff
@@ -110,4 +113,26 @@ def export_bone_flags(self, b_bone, n_node):
                     # default for Div 2 final bones
                     n_node.flags = 0x0196
             else:
-                n_node.flags = 0x0002  # default for Morrowind bones
\ No newline at end of file
+                n_node.flags = 0x0002  # default for Morrowind bones
+
+    def export_lag_bone(self, b_bone, n_node):
+        """Exports a BSLagBoneController for the bone"""
+
+        #BSLagBoneControllers are only for skyrim and later, afaik
+        game = bpy.context.scene.niftools_scene.game
+        if game not in ('SKYRIM'):
```

Comment:
This could now be changed to `if not bpy.context.scene.niftools_scene.is_skyrim():`